input/fsh/SD_Extensions.fsh

```diff
@@ -2,73 +2,48 @@ Extension: ComorbidConditionCode
 Id: mcode-comorbid-condition-code
 Title:  "Comorbid Condition Code"
 Description: "An extension for representing the condition code corresponding to the named comorbid condition."
-* . ^short = "Comorbid Condition Code"
-* . ^definition = "An extension for representing the condition code corresponding to the named comorbid condition."
 * extension 0..0
```

Comment:
```suggestion
```

It looks like SUSHI does set `Extension.extension` to `0..0` without this.

Reply:
That one slipped through. Yes, SUSHI does it for you.